index.js

```diff
@@ -75,8 +82,41 @@ module.exports = function (source, map) {
     }
   });
 
-  function relativePath(fullPath) {
-    return path.relative(rootPath, fullPath);
+  function getAliases (options) {
+    // TODO: Investigate if there is a way to leverage
+    // the Webpack API to get all the aliases
+    var resolveAliases = options.resolve && options.resolve.alias;
+    var resolveLoaderAliases = options.resolveLoader && options.resolveLoader.alias;
+    return Object.assign({}, resolveAliases || {}, resolveLoaderAliases || {});
```

Comment:
Please use the `Object.assign` shim here. Don't want to force a newer Node version when it's not absolutely needed.